### ISSUE TITLE
Issue of editing in vscode

### ISSUE THREAD

**Issue by wszqkzqk** [vala-lang/vala-language-server#279]
**Describe the bug**
LSP fail when typing code in vscode. With:

```
Request textDocument/documentSymbol failed.
```

**Software**
OS and version (e.g. Ubuntu 20.04): Both Arch Linux and Windows 11
Code editor (e.g. VSCode): VSCode
Vala Language Server (e.g. git commit, or PPA/AUR version): 0.48.5
Vala version (`valac --version`): 0.56.3

**To Reproduce**

1. Create a  new vala source file and open it in vscode
2. Type something
3. Showed `Request textDocument/documentSymbol failed.` for many times and failed.

How can I ban it from getting ` textDocument/documentSymbol`?

https://user-images.githubusercontent.com/12961135/212040215-ba343e2e-f23c-4a2c-acd0-db7ee28f932d.mp4

**Comment by DanDankRaft** [comment]
This exact same behavior has been happening to me as well (I think it's a duplicate of #236 )

**Comment by avojak** [comment]
I'm seeing this as well through the VS Code plugin.
Fedora 37
vale-language-server: 0.48.5

**Comment by ppvan** [comment]
Same thing here. I'm tried compile from source but didn't help.
Arch Linux 
```
vala-language-server -v
vala-language-server 0.48.5-15-g7577ffb2
```

**Comment by wszqkzqk** [comment]
It's fixed since v0.48.7
